back/db/migrate/20250930942638_add_min_option_select_to_participation_context.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddMinOptionSelectToParticipationContext < ActiveRecord::Migration[7.1]
+  def change
+    add_column :phases, :voting_min_selected_options, :integer
```

Comment:
```suggestion
    add_column :phases, :voting_min_selected_options, :integer, default: 1, null: false
```

I think it's nicer to just set this to 1 by default. This is also the current behavior, you cannot submit a basket unless you have selected at least one option. And this way you don't have to worry about `nil` values